app/interactors/access_limit/update_interactor.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class AccessLimit::UpdateInteractor < ApplicationInteractor
+  LIMIT_TYPES = AccessLimit.limit_types.values
```

Comment:
Minor, this probably should refer to keys rather than values. Since the key refers to the enum text and the value is what the item is stored as in the DB.

Has no effect for us since we have the same value for key and value.

Reply:
Interesting. I went with 'values' for the same reason. I guess we have different interpretations!